test/integration/targets/ecs_ecr/tasks/main.yml

```diff
@@ -1,16 +1,22 @@
 ---
 - set_fact:
-    ecr_name: '{{ resource_prefix }}-ecr'
+    ecr_name: '{{ resource_prefix | lower }}-ecr'
 
 - block:
 
+    - name: set connection information for all tasks
+      set_fact:
+        aws_connection_info: &aws_connection_info
```

Comment:
This is fine for this PR but we're trying to move the AWS tests to use module_defaults going forward.

Reply:
I don't understand that, and have not changed anything as a result.

Reply:
Just kind of an FYI that we're changing the way we do this, as we'd very much like to have your contributions going forward. Not asking for any changes in this case though cause I totally appreciate how long this PR has dragged on.